docs/use-github-in-bcgov/bc-government-organizations-in-github.md

```diff
@@ -1,120 +1,102 @@
 
-## Working in the open
+We also have the `bcgov-c` organization, where developers can store confidential code.
 
-The [Digital Principles for the Government of B.C.](https://www2.gov.bc.ca/gov/content/governments/policies-for-government/core-policy/policies/im-it-management#12.1.1.5) urge product teams to work in the open as outlined in principle five. GitHub is the leading platform for open-source projects. It allows the Province to collaborate with the open-source community to build software, support innovation and save time and money.
+On behalf of the B.C. Government, the Developer Experience Team manages government GitHub environments. For any questions or concerns, please create a ticket on our [Jira Service Manager](https://citz-do.atlassian.net/servicedesk/customer/portal/2) (JSM).
 
 ## Organizations in GitHub
 
-The Province owns several GitHub organizations, which are described below.
+Recently, we enabled [Single Sign-On (SSO)](#single-sign-on) for `bcgov` and on **August 21, 2024**, we'll enable SSO for `bcgov-c`. The transition helped us understand user needs better, and streamline services.  
+
+Now, we use GitHub Enterprise Cloud, and it hosts the `bcgov` and `bcgov-c` GitHub organizations: 
 
-| Name | Use for... | User Access Requirements | Cost |
-| ---- | ----------- | ----------- | ----- |
-| [bcgov](#bcgov) | Repositories should be hosted in this open repo unless you have strong rationale from your MISO otherwise. | GitHub ID | Free |
-| [bcgov-c](#bcgov-c) | Code you would like to temporarily host in a closed repo, before moving into the bcgov org. | GitHub ID | Covered by OCIO for up to 12 months |
+```mermaid
+  graph TD;
+A[GitHub Enterprise Cloud] --> B[bcgov]
+A --> C[bcgov-c]
+```
 
 ### `bcgov`
 
-The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.
+The `bcgov` organization contains public repositories that hold open-source code or public documents developed by B.C. government teams. This organization is **public**.  To align with our Digital Principles, repositories should be hosted in this open repo. The only exception is strong rationale from your Ministry Security Information Officer.
 
-* Product teams working in the B.C. Gov Private Cloud PaaS OpenShift platform should use this organization unless they are given permission to work in [`bcgov-c`](#bcgov-c).
-* Any member of the `bcgov` organization can create repositories.
-* Existing members of the `bcgov` organization can invite new users to join the organization using the [Just Ask! tool](https://just-ask.developer.gov.bc.ca/). (***Users requesting membership to `bcgov` for themselves will be reviewed and may be declined***) 
+* Product teams working in the B.C. Government Private Cloud PaaS OpenShift platform should use this organization. The exception is permission to work in [`bcgov-c`](#bcgov-c)
```

Comment:
this seems to be a repetition of the above paragraph?

Reply:
@jenreiher-bcgov Does this sentence need to be here? I have no insight into what it's trying to convey, so if it needs removal, I'm alright with it.

Reply:
let's remove!